ModuleConfig.cfc

```diff
@@ -71,7 +71,7 @@ component {
 		//defaults
 		configStruct.autodeploy = {
 			// The tag file location, realtive or absolute
-			"tagFile" : "config/_deploy.tag",
+			"tagFile" : "#modulePath#/config/_deploy.tag",
```

Comment:
Why then does the module come with its own config folder with a copy of the tag inside it?  If this is the case, then you shouldn't have accepted my previous pull either.  I was going off the file structure that ships with the code. 

Also, FYI if you want to have the file in the root, the default behavior of the code is poor as it just throws an error because the file doesn't exist.  I would recommend creating it if it doesn't exist if the module isn't going to look for a file that ships with it.